src/components/Exchange/PositionSeller.js

```diff
@@ -396,11 +393,25 @@ export default function PositionSeller(props) {
         totalFees = totalFees.add(swapFee || bigNumberify(0));
         receiveAmount = receiveAmount.sub(swapFee);
       }
+    }
 
+    convertedReceiveAmount = getTokenAmount(receiveAmount, receiveToken.address, false, infoTokens);
+
+    // Check swap limits (max in / max out)
+    if (allowReceiveTokenChange && swapToken && swapToken.address !== collateralToken.address) {
       const collateralInfo = getTokenInfo(infoTokens, collateralToken.address);
+      const swapTokenInfo = getTokenInfo(infoTokens, swapToken.address);
+
+      const isWrapOrUnwrap =
+        (collateralToken.isNative && swapTokenInfo.isWrapped) || (collateralToken.isWrapped && swapTokenInfo.isNative);
```

Comment:
1. inside contract collateralToken can be one of WBTC, WETH, LINK, UNI (OR WAVAX, WBTC, WETH on Avalanche)
if on contract size position.collateraToken == WETH then in UI position.collateralToken will be `{ isNative: true, address: AddressZero, ... }` (https://github.com/gmx-io/gmx-interface/blob/bc8252b035cf0959f97e61afcad4059b161f1ad0/src/views/Exchange/Exchange.js#L162)
So collateralToken.isWrapper is always `false` (or more precisely is `undefined`)

2. The only possible (from contracts perspective) "wrap-unwrap" operation for position with collateralToken == WETH is to unwrap WETH and send native ETH to user. I'm thinking what is correct check on UI side. at first glance it's `isUnwrap = collateralToken.isNative  && swapTokenInfo.isNative` 🤔. Does `isWrapOrUnwrap` work for you now?